### ISSUE TITLE
Pressing F1 in Error List with MSB3644 error navigates VS to a non helpful page

### ISSUE THREAD

**Issue by rrelyea** [microsoft/msbuild#5498]
### Steps to reproduce

Project file

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

</Project>
```

Command line
```
build in VS 16.6 (before net5.0 tfm support comes in 16.7)
```

### Expected behavior

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3644	The reference assemblies for .NETFramework,Version=v5.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks	n5test	C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets	1177	
```

When I press F1 in VS on that item in the error list, i expect to go to an error page for msb3644.

B) seperetely, would be great to have MSB3644 explain that net5.0 projects need VS 16.7 or later.

### Actual behavior
I get navigated to: https://msdn.microsoft.com/Areas/Epx/Content/500.aspx?aspxerrorpath=/query/dev16.query

```
The site will be moving soon. This content will be permanently redirected.
Unable to Service Request
For the latest headlines and to see what's new, visit the MSDN home page.
```

NuGet supports this...and if i recall correctly, URLs to navigate to might be changeable outside of VS. 
You could start by looking at our code related to: 
```
                    WindowFrameHelper.AddF1HelpKeyword(windowFrame, keywordValue: F1KeywordValuePmUI);
```

### Environment data
`msbuild /version` output:

OS info:
Win10 latest release

**Comment by rainersigwald** [comment]
The plan is to fix this with the new feature added in #5488.

**Comment by rainersigwald** [comment]
Duplicate of #5493
